### DISCUSSION TITLE
Obtainig posts to discussions about TA to e-mail?

### DISCUSSION THREAD

**Discussion by CertHaramas** [orbitalquark/textadept#105] [Q&A]
Hello,
I used to receive posts to TA when it was operating on foica.
I'm not so familiar with GitHub - so elementary question: is it possible to subscribe somewhere to obtain posts about TA to an e-mail? Or do I have to watch the discussions on the GitHub space only (in a web browser)?
Best regards,
Lukas

**Comment by manuelmontenegro** [comment]
Hello, Lukas. You can use the 'Watch' button to receive notifications by email. These notifications include posts to discussions.
